File: send.js
var device_id = '/dev/cu.wchusbserial1420';
var baud_rate = 9600;

var SerialPort = require('serialport');
var port = new SerialPort(device_id, {
	baudRate: baud_rate
});

port.open(function (err) {
  if (err) {
    return console.log('Error opening port: ', err.message);
  }
});

port.on('error', function(err) {
  console.log('Port Error: ', err.message);
});

port.write('Hello\n');

File: send.py
import time
import serial

# configure the serial connections (the parameters differs on the device you are connecting to)
ser = serial.Serial(
    port='/dev/ttyUSB2',
    baudrate=600,
    parity=serial.PARITY_ODD,
    stopbits=serial.STOPBITS_TWO,
    bytesize=serial.SEVENBITS
)

ser.isOpen()

print 'Enter your commands below.\r\nInsert "exit" to leave the application.'

input=1
while 1 :
    # get keyboard input
    #input = raw_input(">> ")
    input="gg"
        # Python 3 users
        # input = input(">> ")
    if input == 'exit':
        ser.close()
        exit()
    else:
        # send the character to the device
        # (note that I happend a \r\n carriage return and line feed to the characters - this is requested by my device)
        ser.write(input )
        #print map(bin,bytearray(input + '\r\n'))